src/components/Footer.tsx

```diff
@@ -42,7 +42,7 @@ const Footer = ({ bgColor = 'background2' }: FooterProps) => {
 							aria-label="Powered by Vercel"
 							isExternal
 						>
-							<Image alt="" src={poweredByVercelLogo} />
+							<Image alt="" src={poweredByVercelLogo as string} />
```

Comment:
SVG files imported within a Next.js project are typed as `any`. I can't override that unless I declare a custom `next-env.d.ts` overriding the types for `*.svg` files. I don't like this idea, since we could miss other type definitions added in the future. Furthermore, I'm casting it to string manually when needed.

More info about this problem here: https://duncanleung.com/next-js-typescript-svg-any-module-declaration/

Reply:
Let's keep things simple and just cast it to `string` manually when needed. We don't have that many SVGs anyways. 😄